BUILDING_ON_WINDOWS.md

```diff
@@ -82,6 +95,19 @@ Note: This installation will take about 200 MB of disk space.
 
 Install [conan 2](https://conan.io/downloads.html) and make sure it's in your `PATH` (default setting).
 
+<details>
+   <summary>How to add conan 2 to PATH</summary>
+   
+1. Type "path" in the start menu and click on the "Edit the system environment variables" result that shows up (this should open up the 'System Properties' window with the 'Advanced' tab selected).
+2. Press on the "Environment Variables..." button near the bottom right of the 'System Properties' window to open up the 'Environment Variables' window.
+3. Within 'Environment Variables', look at the bottom half of the window where it lists 'System variables' and double click on the variable named 'Path' to open up the 'Edit environment variable' window.
+4. Press the 'New' button near the top right to create a new environment variable.
+5. Open up your terminal with the Visual Studio environment variables (e.g. `x64 Native Tools Command Prompt for VS 2022`) and type "where conan" to find the file path (the folder that contains the conan.exe) to add.
+6. Add conan 2's file path (e.g. `C:\Users\willg\AppData\Roaming\Python\Python311\Scripts`) to the blank text box that shows up.
+7. Press the "OK" button.
+
+</details>
+
```

Comment:
Yeah, you're right. I didn't even notice the "default setting". xD 

Also do you know if the 'where conan' command only works if conan 2 is in PATH?

If so, maybe it could be used to verify conan 2 is in PATH.